src/FSharpVSPowerTools.Logic/Linting/OptionsViewModel.fs

```diff
@@ -0,0 +1,215 @@
+            for setting in settings do
+                match setting.Value with
+                | Lines(value) -> 
+                    yield IntViewModel("Lines", value) :> obj
+                | Depth(value) -> 
+                    yield IntViewModel("Depth", value) :> obj
+                | MaxItems(value) -> 
+                    yield IntViewModel("MaxItems", value) :> obj
+                | Length(value) -> 
+                    yield IntViewModel("Length", value) :> obj
+                | MaxCyclomaticComplexity(value) -> 
+                    yield IntViewModel("MaxCyclomaticComplexity", value) :> obj
+                | IncludeMatchStatements(value) -> 
+                    yield BoolViewModel("IncludeMatchStatements", value) :> obj
+                | OneSpaceAllowedAfterOperator(value) -> 
+                    yield BoolViewModel("OneSpaceAllowedAfterOperator", value) :> obj
+                | NumberOfSpacesAllowed(value) -> 
+                    yield IntViewModel("NumberOfSpacesAllowed", value) :> obj
+                | IgnoreBlankLines(value) -> 
+                    yield BoolViewModel("IgnoreBlankLines", value) :> obj
+                | Access(value) -> 
+                    yield AccessViewModel("Access", value) :> obj
+                | Hints(_)
+                | Enabled(_) -> ()
+        ]
+
+    let isRuleEnabled (settings:Map<string, Setting>) =
+        match settings.TryFind "Enabled" with
+        | Some(Enabled(e)) -> e
+        | _ -> false
+
+    let ruleViewModelsFromConfig config = 
+        seq { 
+            for analyser in config.Analysers |> Seq.where (fun x -> x.Key <> "Hints") do 
+                let rules = seq { 
+                    for rule in analyser.Value.Rules do 
+                        yield RuleViewModel(rule.Key,
+                                            [],
+                                            getSettingsViewModelsFromRule rule.Value.Settings,
+                                            IsChecked = isRuleEnabled rule.Value.Settings) 
+                }
+
+                yield RuleViewModel(analyser.Key, 
+                                    rules, 
+                                    getSettingsViewModelsFromRule analyser.Value.Settings,
+                                    IsChecked = isRuleEnabled analyser.Value.Settings) 
+        }
+
+type OptionsViewModel(config:Configuration, files:FileViewModel seq) as this =
+    inherit ViewModelBase()
+
+    let hints = HintsViewModel(config)
+    
+    let selectedRule = this.Factory.Backing(<@ this.SelectedRule @>, None)
+    let newIgnoreFile = this.Factory.Backing(<@ this.NewIgnoreFile @>, "")
+
+    let ignoreFiles =
+        match config.IgnoreFiles with
+        | Some(x) -> 
+            x.Content.Split([|Environment.NewLine|], StringSplitOptions.RemoveEmptyEntries)
+                |> Seq.map (fun x -> x.Trim())
+                |> (fun x -> ObservableCollection<_>(x))
+        | None -> ObservableCollection<string>()
```

Comment:
We have some useful functions over here https://github.com/fsprojects/VisualFSharpPowerTools/blob/master/src/FSharpVSPowerTools.Core/Utils.fs#L349-L391
```fsharp
let ignoreFiles =
    ObservableCollection(
        match config.IgnoreFiles with
        | Some x -> 
            x.Content 
            |> String.split StringSplitOptions.RemoveEmptyEntries [| Environment.NewLine |]
            |> Array.map String.trim
        | None -> [||])
```